lib/portlayer/exec/container.go

```diff
@@ -33,18 +33,16 @@ import (
 	"golang.org/x/net/context"
 )
 
-var containers map[ID]*Container
-var containersLock sync.Mutex
-
 func init() {
-	containers = make(map[ID]*Container)
+	NewContainerCache()
 }
 
 type State int
 
 const (
 	StateRunning = iota + 1
 	StateStopped
+	StateSuspended
```

Comment:
Does this have anything to do with the change set, or has this crept in by mistake?